src/Widgets/calendar/GridDay.vala

```diff
@@ -39,6 +37,8 @@ public class DateTime.Widgets.GridDay : Gtk.EventBox {
      */
     public signal void on_event_add (GLib.DateTime date);
 
+    public Gtk.Grid main_grid;
```

Comment:
Does this need to be public? It seems like other classes shouldn't need to access this

Reply:
Will make it private.